docs/src/architecture/domain-model.md

```diff
@@ -134,6 +135,8 @@ The figure below illustrates the concept of **plugins** and **plugin adapters**.
 
 ### Plugin Adapter
 
+🚫 _Concept of "plugin adapter" not yet implemented_ 🚫
+
```

Comment:
There is nothing to implement here. This done by thin-edge users to drive their child devices.